nanvm-lib/src/interface.rs

```diff
@@ -69,6 +70,56 @@ pub trait Any: PartialEq + Sized + Clone + fmt::Debug {
         // bigint and function
         todo!()
     }
+
+    fn unary_plus(&self) -> Self {
+        if let Some(simple) = self.try_to_simple() {
+            match simple {
+                Simple::Nullish(v) => match v {
+                    Nullish::Null => return Self::new_simple(Simple::Number(0.0)),
+                    Nullish::Undefined => return Self::new_simple(Simple::Number(f64::NAN)),
+                },
+                Simple::Boolean(v) => {
+                    return Self::new_simple(Simple::Number(if v { 1.0 } else { 0.0 }))
+                }
+                Simple::Number(v) => return self.clone(),
+            }
+        }
+        if let Ok(v) = self.clone().try_to::<Self::String16>() {
```

Comment:
I would recommend using the `unpack` function instead. In this case, the match will be simpler.